onnx/test/reference_evaluator_save_test.py

```diff
@@ -0,0 +1,203 @@
+# Copyright (c) ONNX Project Contributors
+
+# SPDX-License-Identifier: Apache-2.0
+# type: ignore
```

Comment:
```suggestion
```

if we want to diable mypy for this file prefer adding the path explicitly to the .lintrunner file’s exclude list under the MYPY section so that we manage them in a single place